lib/devise_two_factor/models/two_factor_authenticatable.rb

```diff
@@ -32,7 +32,8 @@ def validate_and_consume_otp!(code, options = {})
         return false unless code.present? && otp_secret.present?
 
         totp = self.otp(otp_secret)
-        return consume_otp! if totp.verify_with_drift(code, self.class.otp_allowed_drift)
+        # return consume_otp! if totp.verify_with_drift(code, self.class.otp_allowed_drift)
+        return consume_otp! if totp.verify(code, drift_behind: 30)
```

Comment:
```
if totp.verify(code, drift_behind: self.class.otp_allowed_drift, drift_ahead: self.class.otp_allowed_drift)
  return consume_otp! 
end
```

Reply:
Would we be able to use the `otp_allowed_drift` for the drift tolerances?